### PR TITLE
[#4910] Add documentation example for Lint/UnneededDisable

### PR DESCRIPTION
**Pull request by asherkach**
* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by rrosenblum** [comment]
It looks like RuboCop is registering an offense for the example that you provided. Should this be considered a bug in Lint/UnneededDisable?

**Comment by asherkach** [comment]
Ha, no, this is "working as intended". Let me rethink the best way to do an example.

**Comment by asherkach** [comment]
Though disabling `Lint/UnneededDisable` resolves the internal rubocop offense, it also adds the disable and enable to the documentation. Is there a better mechanism to include an example?

File: lib/rubocop/cop/lint/unneeded_disable.rb [hunk 1]
```diff
@@ -12,6 +12,19 @@ module Lint
       # their work. Instead, it waits until it's called in a later stage of the
       # execution. The reason it can't be implemented as a normal cop is that
       # it depends on the results of all other cops to do its work.
+      #
+      # rubocop:disable Lint/UnneededDisable
```
Comment (bbatsov) [inline lib/rubocop/cop/lint/unneeded_disable.rb]: This should go outside the comment, not to affect the documentation.

**Comment by bbatsov** [comment]
You'll also have to rebase on master.

**Comment by asherkach** [comment]
Done (rebased). PTAL.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**